xml/System.Data/DataRowCollection.xml

```diff
@@ -482,7 +482,7 @@
         <AssemblyVersion>4.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Docs>
-        <summary>Gets a specified <see cref="T:System.Data.DataRow" /> using the <see cref="P:System.Data.DataTable.PrimaryKey" /> index (performance should be an O(log n) operation).</summary>
+        <summary>Gets a <see cref="T:System.Data.DataRow" /> using the specified <see cref="P:System.Data.DataTable.PrimaryKey" /> value (performance should be an O(log n) operation).</summary>
```

Comment:
```suggestion
        <summary>Gets a <see cref="T:System.Data.DataRow" /> using the specified <see cref="P:System.Data.DataTable.PrimaryKey" /> value. (Performance should be an O(log n) operation).</summary>
```
nit: but the parenthetical about performance is best in the Remarks section.

Reply:
@MarcioAB can you apply @rpetrusha's suggestion?

I can move the performance note the remarks section on a separate PR.

Reply:
@mairaw,  done, by using the mechanism for small corrections: Edit "directly" the documentation page and create both changes. Change index to value and moved the performance information to remarks section.

Reply:
I don't see the change in this PR. What I was asking you is to click this button on Ron's comment:
![image](https://user-images.githubusercontent.com/12971179/53997124-de277500-40ef-11e9-8394-eb521bdd7063.png)

I can make the Remarks change on a separate PR. If you created a new branch for the changes, we could also create a new PR and discard this one. 

I think you're missing a step when you're trying to suggest content changes for us. Because I see you're creating branches on your fork, but not the PRs. Please feel free to reach out to me for any help!

Reply:
@mairaw I think I can not do that because I do not have "write permissions". The (i) after "Suggested change (i)" says:  This code change can be commited by users with write permissions.

Reply:
@mairaw What I can do is "Approve changes" in the "Review Changes" panel, what I just did.

Reply:
Humm, ok. Something is funky with this PR. I'll start a new one instead.